lib/game.rb

```diff
@@ -12,27 +13,37 @@ def initialize(player_one, player_two, display, messages)
     @messages = messages
   end
 
-  def run
+  def play
     welcome_users
-    @player_one.choose
-    @player_two.choose
-    determine_winner
-    present_winner
+    @player_one.get_input
+    @player_two.get_input
+    result = option_one_result(@player_one.player_input, @player_two.player_input)
+    present_winner(result)
   end
 
   def welcome_users
     @display.present(@messages.welcome_message)
   end
 
-  def determine_winner
-    rules = Rules.new(@player_one.user_input, @player_two.user_input)
-    @winning_option = rules.outcome
+  def option_one_result(option_one, option_two)
+    turn = [option_one, option_two]
+    if option_one == option_two
+      "draw"
+    elsif turn.include?("rock") and turn.include?("scissors") and option_one == "rock"
+      "win"
+    elsif turn.include?("paper") and turn.include?("rock") and option_one == "paper"
+      "win"
+    elsif turn.include?("paper") and turn.include?("scissors") and option_one == "scissors"
```

Comment:
It feels like that you could refactor here with https://sourcemaking.com/refactoring/extract-method

`elsif winner?` (this is a just an example, I'm sure you can find a better name)

Also, the name `option` is quite vague, do you think there is a more descriptive name that you could use?